features/steps/given.py

```
@@ -0,0 +1,18 @@
+from behave import given
+import subprocess
+import time
+
+
+@given("I start the FastAPI server")
+def step_impl(context):
+    # Start the uvicorn server
+    print("Starting FastAPI server")
+    process = subprocess.Popen(
```

Ruff (0.8.0)

`subprocess` call: check for execution of untrusted input

(S603)